e2e/native-esm/staticDataImport.js

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
is this file executed by any e2e test?

Reply:
Yes, in [e2e/native-esm/__tests__/native-esm.test.js](https://github.com/facebook/jest/pull/12392/files#diff-7a98537bdfc98f8a0321f1e556bc226e1eb013e30c266962a788c77df4289a61)